mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
```

Comment:
The `findByIdAndDelete` method returns a promise, but it's not being handled properly. It should be handled using `catch` block to handle any potential errors. Also, it's a good practice to use `async/await` syntax to make the code more readable. 

Instead of:
```
Order.findByIdAndDelete(req.params.orderId) 
  .then(deletedOrder => { 
    if (!deletedOrder) { 
      return res.status(404).json({ 
        error: "Order not found", 
      }); 
    } 
    res.json({ message: "Order deleted successfully" }); 
  }) 
  .catch(error => { 
    res.status(500).json({ 
      error: errorHandler(error), 
    }); 
  });
```
It should be:
```
try {
  const deletedOrder = await Order.findByIdAndDelete(req.params.orderId);
  if (!deletedOrder) {
    return res.status(404).json({
      error: "Order not found",
    });
  }
  res.json({ message: "Order deleted successfully" });
} catch (error) {
  res.status(500).json({
    error: errorHandler(error),
  });
}
```